### PR TITLE
Setup javascript testing

### PR DESCRIPTION
**Pull request by itamargiv**
_**Note:** This PR depends on #61_

This change sets up Jest in order to test our Vue components (or any other JavaScript tests we will need to run in the future).

Bug: [T288063](https://phabricator.wikimedia.org/T288063)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -16,6 +16,22 @@ jobs:
...
+    steps:
+        - uses: actions/checkout@v2
+
+        - name: Setup Node.js
+          uses: actions/setup-node@v1
+          with:
+            node-version: '12.x'
+
+        - name: Install Dependencies
+          run: npm ci
```
Comment (Silvan-WMDE) [inline .github/workflows/test.yml:30]: TIL

**Review by Silvan-WMDE — COMMENTED** [review]
Nice. Worked immediately. See a minor documentation suggestion below.

File: docs/README.md [hunk 2]
```diff
@@ -204,7 +204,13 @@ To manually compile assets for your local dev environment run:
 
 ```
 sail npm run dev
-```  
+```
+
+To run any Javascript tests created:
```
[the remark this answers is not in the document]
Comment (itamargiv) [reply docs/README.md:209]: :+1: Great idea!

**Review by guergana — APPROVED** [review]
Looks pretty good i would say. :+1:

**Review by Silvan-WMDE — APPROVED** [review]
Tests are set up and pass, developer docs are updated, GitHub action runs the test - AC's fully met. :100:

**Pull request merged**